File: src/ai-user-response/db_connection.py
# db_connection.py
from mongoengine import connect
import os

def get_db_connection():
    db_uri = os.getenv('MONGODB_URI', 'mongodb://localhost:27017/myAIDatabase')
    return connect(host='mongodb+srv://adityadubey:1234@cluster0.bzu5sg9.mongodb.net/')

# You can call this function in other modules to establish a DB connection

File: src/server/database/dbConnection.js
const mongoose = require('mongoose');
require('dotenv').config();

const connectDB = async () => {
  try {
    await mongoose.connect('mongodb+srv://adityadubey:1234@cluster0.bzu5sg9.mongodb.net/', {
      useNewUrlParser: true,
      useUnifiedTopology: true
    });
    // await mongoose.connect(process.env.MONGODB_URI, {
    //   useNewUrlParser: true,
    //   useUnifiedTopology: true
    // });
    console.log('MongoDB Connected...');
  } catch (error) {
    console.error('Database connection error:', error);
    process.exit(1);
  }
};

module.exports = connectDB;
